### ISSUE TITLE
speciallink prefixes

### ISSUE THREAD

**Issue by y-ack** [12Me21/markup2#3]
presumably, allow qcs: in addition to sbs:

it also might be desirable to make modifying accepted special prefixes easier; the aucep had at one point proposed imageboard shorthands.

**Comment by 12Me21** [comment]
yeah, probably want more than just `\w+:` prefix though, since that's going to be triggered accidentally too much
